docs/pages/connect-your-client/vnet.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Using VNet
+description: Using VNet
+---
+
```

Comment:
I would add a quick intro paragraph to help users determine if they should use this guide—maybe use the first paragraph of "How it works", then add a sentence saying something like, "This guide explains how to use VNet"?

Reply:
I reworded "How it works" to avoid mentioning earlier Teleport versions. Now it's pretty much a single paragraph. Do you think an intro section is still necessary?

Reply:
I've suggested a very quick one. I think all guides should have some kind of initial paragraph to orient the user and help them make sure they're in the right place